var/spack/repos/builtin/packages/utf8/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Utf8(Package):
```

Comment:
Yes, it could be, I even had it that way at one point while working on this. I settled on `utf8` as that is what the zip file is called. The `spack create` command determined the name was `utf` so this one is odd.

Reply:
@scheibelp Can you clarify whether this should be `utf8-cpp` or `utf8cpp`? I ask because the naming for this package is all over the map. The project homepage refers to `utf8-cpp`. Other than that, the code for the software is referred to as:

- utfcpp
- utf8cpp
- utf8

The VTK project seems to prefer `utf8cpp` so that is what I would lean towards.

Reply:
I prefer `utf8-cpp` (to achieve that, you would choose `Utf8Cpp` as the name of the Package class in the `package.py` file).